examples/Example1.elm

```diff
@@ -82,10 +82,10 @@ view model =
 dropdownConfig : Dropdown.Config String Msg
 dropdownConfig =
     let
-        itemToPrompt item =
-            Element.text item
+        itemsToPrompt items =
+            Element.text <| Maybe.withDefault "nothing selected" <| List.head items
```

Comment:
Niptick: Personally I prefer to rever the pipe and break lines, something like this:

```elm
List.head items
    |> Maybe.withDefault "nothing selected"
    |> Element.text
```

I think this makes it easy to follow and understand quickly, but like I said it's a personal thing and it's totally up to you